### PR TITLE
Tickets/dm 2900

### PR DESCRIPTION
**Pull request by fjammes**

### CODE REVIEW

File: datasets/case02/queries/1002.2_coneMagColor.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+-- Cone-magnitude-color search
+-- See http://dev.lsstcorp.org/trac/wiki/dbQuery003
```
Comment (andy-slac) [inline datasets/case02/queries/1002.2_coneMagColor.sql]: This page does not exist, we should either replace this link with better one or remove it, this applies to all new files

File: python/lsst/qserv/tests/benchmark.py [hunk 2]
```diff
@@ -177,6 +174,13 @@ def _parseFile(self, qF, withQserv):
             elif withQserv and line.startswith("-- withQserv"):
                 # strip the "-- withQserv" text
                 qText.append(line[13:])
+            elif line.endswith("-- noQserv"):
+                if withQserv:
+                    # skip this line
+                    pass
+                else:
+                    # strip the "-- noQserv" text
+                    qText.append(line[:-10])
```
Comment (andy-slac) [inline python/lsst/qserv/tests/benchmark.py]: This replacement does not do exactly the same thing. Original regex search allowed comments like:

    SOME SQL -- noQserv (some comment here)

while this replacement does not allow anything after `noQserv`. Not sure if this matters, I presume that you checked that there is no comments like above in our existing tests.

File: datasets/case02/queries/1002.2_coneMagColor.sql (hunk 1 above)
Comment (fjammes) [inline datasets/case02/queries/1002.2_coneMagColor.sql]: Good it, it has been renamed to https://dev.lsstcorp.org/trac/wiki/db/queries/003

File: python/lsst/qserv/tests/benchmark.py (hunk 2 above)
Comment (fjammes) [inline python/lsst/qserv/tests/benchmark.py]: Yes, test passes. I think that "-- noQserv" should start the line, like other pragmas

Comment (andy-slac) [inline python/lsst/qserv/tests/benchmark.py]: I think the reason why "-- noQserv" is at the end of the line and "-- withQserv" at the beginning is that you can feed this file directly to mysql without any replacements, i.e. it looks like correct mysql query (but not qserv query).

**Comment by fritzm** [comment]
Closing merge orphan

**Pull request closed without merging**